operations/mimir/autoscaling.libsonnet

```diff
@@ -66,7 +66,7 @@
   // metricWithWeight will multiply the metric by the weight provided if it's different from 1.
   local metricWithWeight(metric, weight) =
```

Comment:
Maybe out of scope, but when reviewing this PR it seems to me that this function should be called `queryWithWeight` and the `metric` arg should be `query`, since in fact a query is modified. It's much less confusing to the reviewer when they know the function acts on a query :)